ui/app/AppLayouts/Chat/stores/RootStore.qml

```diff
@@ -555,7 +555,13 @@ QtObject {
     }
 
     // Needed for TX in chat for stickers and via contact
-    readonly property var accounts: walletSectionAccounts.accounts
+    readonly property var accounts: SortFilterProxyModel {
+        sourceModel: walletSectionAccounts.accounts
+        filters: ValueFilter {
+            roleName: "canSend"
+            value: true
+        }
+    }
```

Comment:
I have no idea if this is still used. I couldn't find the place in the code, but search for `accounts` is not easy.

Reply:
Ah, so my change is probably not needed, since we already have the `SortFilterProxyModel`

Reply:
In a different context though